src/components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed.js

```diff
@@ -429,6 +429,32 @@ class ActivityFeed extends React.Component<Props, State> {
         this.updateFeedItems(comments, tasks, versions, file);
     }
 
+    /**
+     * Adds a versions entry if the current file version was restored from a previous version
+     *
+     * @param {FileVersions} versions - API returned file versions for this file
+     * @return {FileVersions} modified versions array including the version restore
+     */
+    updateRestoredVersions(versions: FileVersions) {
```

Comment:
addRestoredVersion? You're only updating one version here.

Reply:
Added tests